frontend/src/scenes/experiments/experimentLogic.tsx

```diff
@@ -1261,6 +1261,53 @@ export const experimentLogic = kea<experimentLogicType>([
                     return (variantResults[variantResults.length - 1].count / variantResults[0].count) * 100
                 },
         ],
+        credibleIntervalForVariant: [
+            (s) => [s.experimentInsightType],
+            (experimentInsightType) =>
+                (
+                    experimentResults:
+                        | Partial<ExperimentResults['result']>
+                        | CachedExperimentFunnelsQueryResponse
+                        | CachedExperimentTrendsQueryResponse
+                        | null,
+                    variantKey: string
+                ): [number, number] | null => {
+                    const credibleInterval = experimentResults?.credible_intervals?.[variantKey]
+                    if (!credibleInterval) {
+                        return null
+                    }
+
+                    if (experimentInsightType === InsightType.FUNNELS) {
```

Comment:
The issue is here - you're checking the insight type of the main metric, not of the current secondary metric.

This probably worked for you because your main goal was Funnel, but was breaking for me because my main metric was Trend.

You can get the secondary metric type like so:
`const experimentInsightType = experimentResults?.filters?.insight`

Reply:
37d9cff9d018a78758a322e75d61b4beba3352f0